static/files/download-data-for-wlan-firmwares.bat

```diff
@@ -20,21 +20,28 @@ if /i {%a4%}=={y} goto :yes
 if /i {%a4%}=={yes} goto :yes
 set HAIKU_TEMP=system\data\firmware
 :yes
-set baseURL=http://cgit.haiku-os.org/haiku/plain/src/system/libroot/posix
+set baseURL=https://raw.githubusercontent.com/haiku/firmware/master/wifi
+set haikuPortsURL=https://eu.hpkg.haiku-os.org/haikuports/master/x86_64/current/packages
 
 rem Make the temp directories to store files
-mkdir wifi-firmware\%HAIKU_TEMP%\broadcom43xx\b43-fwcutter\bits
+mkdir wifi-firmware\%HAIKU_TEMP%\iprowifi2100
+mkdir wifi-firmware\%HAIKU_TEMP%\iprowifi2200
+mkdir wifi-firmware\%HAIKU_TEMP%\broadcom43xx\b43-fwcutter
 mkdir wifi-firmware\%HAIKU_TEMP%\marvell88w8335
 
+cd wifi-firmware\%HAIKU_TEMP%\iprowifi2100
+wget %baseURL%/intel/ipw2100-fw-1.3.tgz
+
+cd wifi-firmware\%HAIKU_TEMP%\iprowifi2200
+wget %baseURL%/intel/ipw2200-fw-3.1.tgz
+
 cd wifi-firmware\%HAIKU_TEMP%\broadcom43xx
-wget http://downloads.openwrt.org/sources/wl_apsta-3.130.20.0
+wget %baseURL%/b43/wl_apsta-3.130.20.0
 cd b43-fwcutter
-wget http://bues.ch/b43/fwcutter/b43-fwcutter-019.tar.bz2
-wget %baseURL%/glibc/string/byteswap.h
-cd bits
-wget %baseURL%/glibc/include/arch/x86/bits/byteswap.h
+wget %haikuPortsURL%/packages/b43_fwcutter-019-2-x86_64.hpkg

```

Comment:
this is specific to x86_64 as well (and same limitation in the .bat file)

Reply:
I think the OpenBSD driver for this hardware does not need fwcutter. Perhaps we can switch to that. However, I don't know if anyone actually has the hardware in question anymore to test with?

Reply:
Should I just remove this driver then and update the firmware install script? Or does it make sense to remove these scripts entirely? In the end the driver I was hoping would work didn't but I thought I would fix them for the next person.

Reply:
Which driver is that? What is your PCI device? I may be able to switch to an OpenBSD driver which may work better than the FreeBSD ones.

Reply:
Should we make the warnings on the website about this script being useful only for extremely obsolete hardware larger? Can we include a cutoff year? Something like "if your wifi card is from after 2006, you don't need this"?

Reply:
@waddlesplash can you clarify how OpenBSD avoids fwcutter? If I remember correctly, the issue is that the firmware is built inside the Windows driver and the license does not allow redistribution of modified versions. Did they implement fwcutter inside their driver, or did they just ignore the license and provide the bare firmware file?

Reply:
I think they implement something akin to fwcutter inside the driver, yes.

Reply:
> Which driver is that? What is your PCI device? I may be able to switch to an OpenBSD driver which may work better than the FreeBSD ones.

Marvell 88W8897 [AVASTAR] 802.11ac - Let me know if you need anything else.

Reply:
> Should we make the warnings on the website about this script being useful only for extremely obsolete hardware larger? Can we include a cutoff year? Something like "if your wifi card is from after 2006, you don't need this"?

I like this. Let me try to put some wording together. Might have saved me time.

Reply:
> Marvell 88W8897 [AVASTAR] 802.11ac

The drivers this firmware is for all predate 802.11n, much less 802.11ac. There are no FreeBSD or OpenBSD drivers for your hardware, as far as I know.